shrinkwrap-lxc.sh

```diff
@@ -9,7 +9,7 @@ sleep 2
 
 function dependencies {
   lxc exec shrinkwrap -- apt update
-  lxc exec shrinkwrap -- apt install -y python3-pip unzip docker.io
+  lxc exec shrinkwrap -- apt install -y python3-pip docker.io
   lxc exec shrinkwrap -- pip install -r /root/requirements.txt
   lxc exec shrinkwrap -- snap install juju --classic
```

Comment:
We can drop this too, right?

Reply:
drop docker?  I think i'm still using `docker pull/docker save` to create `.tgz` images of the containers

If there's another way i should pull them to tgz without using docker -- that'd be nice.

Reply:
Done